### PR TITLE
{bio}[GCCcore/12.3.0] gemmi v0.6.5

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/gemmi/gemmi-0.6.5-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+local_pipcmd += ' --config-settings="cmake.args=-DINSTALL_DEV_FILES=OFF"'
+local_pipcmd += ' --config-settings=editable.rebuild=true -Cbuild-dir=%(builddir)s/pybuild '
+postinstallcmds = [
+    (local_pipcmd)
+]
+
+sanity_check_paths = {
+    'files': ['bin/gemmi', 'lib/libgemmi_cpp.%s' % SHLIB_EXT, 'lib/libgemmi_cpp.a'],
+    'dirs': ['bin', 'lib', 'include/gemmi', 'lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s.dist-info']
+}
```
Comment (smoors) [inline easybuild/easyconfigs/g/gemmi/gemmi-0.6.5-GCCcore-12.3.0.eb:55]: @ThomasHoffmann77 is it possible to add a `sanity_check_commands` here?

File: easybuild/easyconfigs/g/gemmi/gemmi-0.6.5-GCCcore-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+local_pipcmd = "mkdir %(builddir)s/pybuild &&"
+local_pipcmd += "python -m pip install -ve %(builddir)s/%(name)s-%(version)s"
+local_pipcmd += " --no-deps --ignore-installed --prefix=%(installdir)s --no-build-isolation"
+local_pipcmd += ' --config-settings="cmake.args=-DSTANDALONE_PYTHON_MODULE=OFF"'  # use lib/libgemmi_cpp.so
+local_pipcmd += ' --config-settings="cmake.args=-DBUILD_GEMMI_PROGRAM=OFF"'
+local_pipcmd += ' --config-settings="cmake.args=-DINSTALL_DEV_FILES=OFF"'
+local_pipcmd += ' --config-settings=editable.rebuild=true -Cbuild-dir=%(builddir)s/pybuild '
+postinstallcmds = [
+    (local_pipcmd)
+]
```
Comment (smoors) [inline easybuild/easyconfigs/g/gemmi/gemmi-0.6.5-GCCcore-12.3.0.eb:50]:
```suggestion
postinstallcmds = [local_pipcmd]
```

File: easybuild/easyconfigs/g/gemmi/gemmi-0.6.5-GCCcore-12.3.0.eb (hunk 1 above)
Comment (ThomasHoffmann77) [reply easybuild/easyconfigs/g/gemmi/gemmi-0.6.5-GCCcore-12.3.0.eb:53]: Done

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**